Resources/Prototypes/Entities/Structures/Decoration/bonfire.yml

```
@@ -31,6 +31,9 @@
     sound:
       path: /Audio/Ambience/Objects/fireplace.ogg
   - type: AlwaysHot
+  - type: Construction # Frontier
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)